src/core/router/net_db/impl.h

```diff
@@ -197,16 +197,16 @@ class NetDb : public NetDbTraits {
 
   std::shared_ptr<const RouterInfo> GetClosestFloodfill(
       const IdentHash& destination,
-      const std::set<IdentHash>& excluded) const;
+      const std::set<IdentHash>& excluded);
```

Comment:
These getters should remain const. They should only be accessing members, not mutating.

Reply:
> These getters should remain const

I thought so as well, but didn't know how much work would be needed on the dispatcher to make it const as well. Will work on this.

Reply:
Not much to it:

```diff
diff --git a/src/core/util/exception.cc b/src/core/util/exception.cc
index 936dfa9f..4bc5ad23 100644
--- a/src/core/util/exception.cc
+++ b/src/core/util/exception.cc
@@ -45,7 +45,7 @@ namespace core {
 Exception::Exception(const char* message) : m_Message(message) {}
 
 // TODO(anonimal): exception error codes to replace strings?
-void Exception::Dispatch(const char* message)
+void Exception::Dispatch(const char* message) const
 {
   // Message to log
   std::string log;
diff --git a/src/core/util/exception.h b/src/core/util/exception.h
index dc84d926..e7ae5cc1 100644
--- a/src/core/util/exception.h
+++ b/src/core/util/exception.h
@@ -49,7 +49,7 @@ class Exception final {
   /// @brief Exception class dispatcher
   /// @details Set optional exception message, concats messages, adds trivial formatting
   /// @param message String message to log for exception
-  void Dispatch(const char* message = "");
+  void Dispatch(const char* message = "") const;
 
  private:
   std::string m_Message;
```

Reply:
Wow, really not much. Will make the changes.